admin/bsf-analytics/class-bsf-analytics-stats.php

```diff
@@ -243,7 +243,8 @@ function wp_timezone_string() {
 			return $timezone_string;
 		}
 
-		$offset  = (float) get_option( 'gmt_offset' );
+		$offset  = get_option( 'gmt_offset' );
+		$offset  = is_numeric( $offset ) ? (float) $offset : 0;
```

Comment:
Why we are converting the $offeset to `float` and in the next line in `Int` ? Either convert it into `float` or in `int`.

Reply:
Changes are done